### PR TITLE
feature: Fixed all allowed Composer dependencies and enabled CI jobs to test against boundary conditions

### PR DESCRIPTION
**Pull request by piotrkreft**
# Description

Issue #1174 
## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

- [x] Unit Tests
- [x] Behat Test

# Checklist:
- [x] I have read the contribution requirements and fulfill them.
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by rprzedzik — APPROVED** [review]

File: module/account/features/account.feature [hunk 1]
```diff
@@ -792,14 +792,14 @@ Feature: Account module
       """
     Then the response status code should be 400
 
-  Scenario: Create user (long mail)
+  Scenario: Create user (long mail - 64 chars local part)
```
Comment (piotrkreft) [inline module/account/features/account.feature:795]: A newer versions of `egulias/email-validator` used both within Symfony validators and Ergonode VO's allow only 64 chars in email part. This is valid according to RFC therefore adjusting to it.

**Review by rprzedzik — APPROVED** [review]

**Review by wiewiurdp — APPROVED** [review]
